qemu/tests/nic_hotplug.py

```diff
@@ -50,6 +49,9 @@ def run(test, params, env):
     nic_hotplugged = []
     try:
         for nic_index in xrange(1, nic_hotplug_count + 1):
+            # need to reconnect serial port after
+            # guest reboot for windows guest
+            session = vm.wait_for_serial_login(timeout=login_timeout)
```

Comment:
Hi @xutian 

Do we need judge the guest which is windows or linux here? 

Thanks.